creme/creme_core/static/creme_core/js/tests/views/search.js

```diff
@@ -555,14 +568,23 @@ QUnit.test('creme.search.SearchBox.keys (escape => close popover)', function() {
 
     element.find('input[type="text"]').trigger('focus');
 
-    equal(true, search.isOpened());
-    equal(1, element.find('.inline-search-results.showing').length);
-    equal(1, $('.glasspane').length);
+    stop(2);
 
-    element.find('input[type="text"]').trigger($.Event("keydown", {keyCode: 27}));
+    setTimeout(function() {
+        equal(true, search.isOpened());
+        equal(1, element.find('.inline-search-results.showing').length);
+        equal(1, $('.glasspane').length);
+        start();
+
+        element.find('input[type="text"]').trigger($.Event("keydown", {keyCode: 27}));
+
+        setTimeout(function() {
+            equal(false, search.isOpened());
+            equal(0, element.find('.inline-search-results.showing').length);
+            equal(0, $('.glasspane').length);
+            start();
+        }, 100);
+    }, 100);
 
-    equal(false, search.isOpened());
-    equal(0, element.find('.inline-search-results.showing').length);
-    equal(0, $('.glasspane').length);
 });
 }(jQuery));
```

Comment:
In a separated commit (same PR)?

Reply:
It was a hack to try to fix random issues that make the CI fails on hugo's branches but this one too. I can create a separate commit, np.